biopandas/mmcif/mmcif_parser.py

```diff
@@ -0,0 +1,444 @@
+# The CIF parser relieves heavily on the parser from PDBJapan (author: Gert-Jan Bekker).
+# Python CIF parser: https://gitlab.com/pdbjapan/tools/cif-parsers
+# License: MIT
+#   See https://gitlab.com/pdbjapan/tools/cif-parsers/blob/master/LICENSE
+import gzip
+import re
+
+try:
+    import json
+except ImportError:
+    import simplejson as json
+
+try:
+    str.partition
+
+    def partitionString(string, sep):
+        return string.partition(sep)
+
+
+except:
+
+    def partitionString(string, sep):
+        tmp = string.split(sep)
+        return [tmp.pop(0), sep, sep.join(tmp)]
+
+
+class _loop:
```

Comment:
Shouldn't we use camel (or at least first-letter uppercase) for this to be consistent with Python style?

Reply:
E.g., `_Loop`

Reply:
In general, I'd agree. Although In this case, `_loop` is a [directive in the mmCIF format](https://mmcif.wwpdb.org/docs/tutorials/mechanics/pdbx-mmcif-syntax.html) so I think it might actually make it more readable at the expensive of being truly pythonic. I don't have particularly strong feelings either way.

Reply:
Oh I see. So was there an intention to keep this a private class? Because the current code is `_loop` and the format specifies the `loop_` directive.

In any case, I think that `Loop_`, `LoopDir` or `LoopMMCIF` would be better to make sure it's clear that it's a class.